Resolving Issue #2572 (#2586)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

- Microsoft Teams 
- Microsoft Audio Conferencing 

> > [!NOTE]

Reviewer: ```suggestion
```

> > When you are escalating from a 1:1 call the audio conferencing license is required

Reviewer: ```suggestion
> -  To escalate from 1:1 to group call you need an Audio Conferencing license.
> -  The E5 license includes both Phone System and Audio Conferencing.
```

> > When you are escalating from a 1:1 call the audio conferencing license is required

Reviewer: ```suggestion
```

> [!IMPORTANT]

> >  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.

Reviewer: ```suggestion
>  The Audio Conferencing license is *required* to: 
> - Escalate from a 1:1 call to a group call.
> - Add external participants to scheduled meetings, by either dialing out or providing the dial-in number. 
```

> > [!NOTE]

Reviewer: ```suggestion
```

> > The E5 license includes both Phone System and Audio Conferencing.   

Reviewer: ```suggestion
```
